### PR TITLE
Fix for is_fuzzy

### PR DESCRIPTION
**Pull request by phlax**
currently unit state is not being correctly changed (and bad subs created) when un/checking the fuzzy flag.

This pr should resolve that

### CODE REVIEW

File: pootle/apps/pootle_store/forms.py [hunk 1]
```diff
@@ -262,13 +239,16 @@ def clean(self):
...
+            not_cleared = (
+                self.request is not None
+                and not check_permission('administrate', self.request)
+                and is_fuzzy)
+            if not_cleared:
+                self.add_error(
+                    'is_fuzzy',
+                    forms.ValidationError(
+                        _('Needs work flag must be '
+                          'cleared')))
```
Comment (unho) [inline pootle/apps/pootle_store/forms.py]: Nit: at least the translatable string should fit in the same line.

Comment (phlax) [inline pootle/apps/pootle_store/forms.py]: yep, i think it was copypasta - ill update

File: pootle/apps/pootle_store/forms.py [hunk 2]
```diff
@@ -655,11 +635,16 @@ def save(self):
...
-        label=_('Needs work'))
+    is_fuzzy = forms.BooleanField(
+        initial=False,
+        label=_("Needs work"))
     target_f = MultiStringFormField(required=False)
 
+    def clean_is_fuzzy(self):
+        if self.data["is_fuzzy"] == "0":
+            return False
+        return True
```
Comment (unho) [inline pootle/apps/pootle_store/forms.py]: Simpler:

```python
        return self.data["is_fuzzy"] != "0"
```

Comment (phlax) [inline pootle/apps/pootle_store/forms.py]: yep

**Comment by unho** [comment]
Reviewed and tested. gtm

**Pull request merged**